### PR TITLE
Formatting and heading levels

### PR DESCRIPTION
**Pull request by pcuenca**
(The heading changes are unrelated but should help with the automatic ToC)

### CODE REVIEW

File: tgi-benchmarking.md [hunk 1]
```diff
@@ -4,14 +4,15 @@ thumbnail: /blog/assets/tgi-benchmarking/tgi-benchmarking-thumbnail.png
 authors:
 - user: derek-thomas
 ---
-# Introduction
+
+# Benchmarking Text Generation Inference
```
Comment (pcuenca) [inline tgi-benchmarking.md:8]: The title is not rendered automatically, we usually write it in a `#` header

File: tgi-benchmarking.md [hunk 2]
```diff
@@ -4,14 +4,15 @@ thumbnail: /blog/assets/tgi-benchmarking/tgi-benchmarking-thumbnail.png
 authors:
 - user: derek-thomas
 ---
-# Introduction
+
```
Comment (pcuenca) [inline tgi-benchmarking.md:7]: I'm suspicious of this missing blank space

**Review by datavistics — APPROVED** [review]
Looks good

**Comment by datavistics** [comment]
Trying to find web editor

**Pull request merged**